src/theme/theme.tsx

```diff
@@ -32,7 +32,10 @@ const colors = {
 
 export const theme = extendTheme({ colors });
 
-export function getColorByStatus(status: string | undefined): string {
+export function getColorByStatus(
+  status: string | undefined,
+  defaultColor: string
+): string {
```

Comment:
This function is too specific to be placed in `./theme/theme.tsx`, please put it closer to where it is used, e.g. in `./src/views/wiki/immunizationWiki.util.ts`.

Reply:
That would lead to code duplication as it is a function that is used in the wiki, on the dashboard and on the details pages. Are you sure that we should move it?

Reply:
Makes sense 👍

Reply:
You can also check but giving the method a string not conforming to the type definition does not throw any error even though would expect it to throw one. Right?

Reply:
You should be getting a compile-time error, not runtime (Typescript only exists during compilation, during runtime none of the types are present anymore, it's just plain Javascript). Let me check.